models/articles-model.js

```diff
@@ -55,11 +55,27 @@ exports.selectArticles = (sort_by = "created_at", order = "desc", topic) => {
 };
 
 exports.selectArticleById = (article_id) => {
-  let queryStr = `
-    SELECT * FROM articles WHERE article_id = $1;`;
+  let queryStr = `SELECT 
+    articles.created_at,
```

Comment:
have a look at other ways of selecting everything from one table but not another :)